README.md

```diff
@@ -1,59 +1,29 @@
 # sparkmagic
-IPython magics for working with remote Spark clusters
 
-## Development team and Advocate
+Sparkmagic is a set of tools for interactively working with remote Spark clusters through [Livy](https://github.com/cloudera/hue/tree/master/apps/spark/java), a Spark REST server, in [Jupyter](http://jupyter.org) notebooks.
+The Sparkmagic project includes a set of IPython magics for interactively running Spark code in multiple languages, as well as some kernels that you can use to turn Jupyter into an integrated Spark environment.
```

Comment:
Some sections I can think of:

* architecture section: pros/cons
* contributing section
* build status/license/etc table like the one at [Pandas repo](https://github.com/pydata/pandas)
* useful links (spark docs, livy)